src/config/browse-by-config.interface.ts

```diff
@@ -0,0 +1,7 @@
+import { Config } from './config.interface';
+
+export interface BrowseByConfig extends Config {
```

Comment:
Could we have some TypeDocs added here, please.  Especially since the `oneYearLimit`, `fiveYearLimit` fields below are not self-explanatory, so it'd be good to describe the purpose of this interface